opm/material/fluidmatrixinteractions/EclEpsTwoPhaseLaw.hpp

```diff
@@ -469,14 +469,8 @@ class EclEpsTwoPhaseLaw : public EffLawT::Traits
     static Evaluation unscaledToScaledPcnw_(const Params& params, const Evaluation& unscaledPcnw)
     {
         if (params.config().enableLeverettScaling()) {
-            // HACK: we need to divide the "unscaled capillary pressure" by the
-            // conversion factor for deck to SI pressures because this quantity is
-            // actually *not* a pressure but the (dimensionless) value of the
-            // J-function. Since opm-parser currently always treads this column as a
-            // pressure, it multiplies it with the conversion factor and we have to
-            // revert this here.
             Scalar alpha = params.scaledPoints().leverettFactor();
-            return unscaledPcnw*(alpha/params.config().deckPressureConversionFactor());
+            return unscaledPcnw*alpha;
```

Comment:
not directly; it is the square root of the applicable permeability divided by the square root of porosity times a constant times surface tension:  https://github.com/OPM/opm-material/blob/master/opm/material/fluidmatrixinteractions/EclEpsScalingPoints.hpp#L378